### PR TITLE
Do not update parallel subjobs if deleted

### PR DESCRIPTION
**Pull request by ttstarck**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,10 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :development do
+  gem 'appraisal'
+  gem 'rspec', '~> 3.10.0'
+  gem 'rake'
+end
```
Comment (ttstarck) [inline Gemfile:9]: Moved these from the gemspec to the gemfile

File: lib/sidekiq/superworker/subjob_processor.rb [hunk 2]
```diff
@@ -110,7 +110,9 @@ def descendants_are_complete(subjob)
             if siblings_descendants_are_complete
               Superworker.debug "#{subjob.to_info}: Parent (#{parent.to_info}) is complete"
               descendants_are_complete(parent)
-              parent.update_attribute(:status, 'complete') if is_child_of_parallel
+              if is_child_of_parallel && !Superworker.options[:delete_subjobs_after_superjob_completes]
```
Comment (ttstarck) [inline lib/sidekiq/superworker/subjob_processor.rb:113]: If we have delete subjobs enabled, we will delete the parent subjob but then write the job again to redis saying its complete. This should catch that edgecase.

File: sidekiq-superworker.gemspec [hunk 3]
```diff
@@ -4,19 +4,16 @@ Gem::Specification.new do |s|
...
   s.description = s.summary = %q{Define dependency graphs of Sidekiq jobs}
-  s.homepage      = 'https://github.com/socialpandas/sidekiq-superworker'
+  s.homepage      = 'https://github.com/Invoca/sidekiq-superworker'
 
   s.files         = Dir['{lib}/**/*'] + ['MIT-LICENSE', 'Rakefile', 'README.md']
   s.name          = 'sidekiq-superworker'
   s.require_paths = ['lib']
   s.version       = Sidekiq::Superworker::VERSION
   s.license       = 'MIT'
+  s.metadata["allowed_push_host"] = "https://gem.fury.io/invoca"
```
Comment (ttstarck) [inline sidekiq-superworker.gemspec:14]: I didn't want to deal with changing the gemname so I just used a new invoca source... Maybe we should just change the name 🤷

Comment (primiti) [reply sidekiq-superworker.gemspec:14]: Ideally we can push this back upstream, so I would prefer not to update the name.

**Review by primiti — APPROVED** [review]
Awesome.  Thank you.

**Pull request merged**